src/main/java/core/basesyntax/Plane.java

```diff
@@ -6,6 +6,75 @@
  * которые могут потребоваться для реализации паттерна Builder.</p>
  */
 
-public class Plane {
+public final class Plane {
```

Comment:
what is the purpose of `final` keyword here?

Reply:
We are not extending the class anymore

Reply:
in this case no, but you should write code so that it could be reusable in the future
what if we needed to extend it one day? there is no reason why we should not allow this

Reply:
You're right. Done